README.md

```diff
@@ -1,3 +1,21 @@
 # openstack-must-gather
 
 OCP must-gather scripts for osp operator logs/data collection.
+
+## Usage
+
+(optional) Build and push the must-gather image to a registry:
+
+```
+git clone ssh://git@github.com/openstack-k8s-operators/osp-director-operator-must-gather.git
+cd osp-director-operator-must-gather
+IMAGE_TAG=<tag> IMAGE_REGISTRY=<registry> MUST_GATHER_IMAGE=must-gather make
+```
+
+On a machine where you have `oc adm` access, do the following:
+
```

Comment:
Maybe we can add a comment about making the image public in the registry if the user is building his own must-gather image.